src/translations/pl/translation.json

```diff
@@ -260,20 +260,20 @@
     "Chance": "Szansa",
     "Count": "Ilość",
     "map": "mapa",
-    "Spawn chance": "Szansa pojawienia",
+    "Spawn chance": "Szansa pojawienia się",
     "Chance that the boss spawns on a given map": "Szansa, że boss pojawi się na danej mapie",
-    "Health": "Życie",
+    "Health": "PŻ",
     "Total boss health": "Łączne PŻ bossa",
-    "patrol": "patroluje",
-    "rush": "naciera",
-    "stalker": "prześladowca",
-    "hostile and accurate": "wrogi i celny",
-    "patrol and highly armored": "patroluje i ciężko uzbrojony",
-    "group patrol": "patroluje w grupie",
-    "frequent healing and stim injections": "częste leczenie i zastrzyki stymulujące",
-    "sniper": "snajper",
-    "batshit insane": "cholernie szalony",
-    "Behavior": "Zachpowanie",
+    "patrol": "Patroluje",
+    "rush": "Naciera",
+    "stalker": "Prześladowca",
+    "hostile and accurate": "Wrogi i celny",
+    "patrol and highly armored": "Patroluje i ciężko uzbrojony",
+    "group patrol": "Patroluje w grupie",
+    "frequent healing and stim injections": "Częste leczenie i zastrzyki stymulujące",
+    "sniper": "Snajper",
+    "batshit insane": "Cholernie szalony",
```

Comment:
I know that @GrantBirki created it. We should ask him :)

@GrantBirki do you agree to make this First letter uppercase,

Reply:
Fine with me!